src/Domain/DomainCreators/RotatedIntervals.hpp

```diff
@@ -41,15 +46,15 @@ class RotatedIntervals : public DomainCreator<1, TargetFrame> {
     using type = std::array<bool, 1>;
     static constexpr OptionString help = {
         "Sequence for [x], true if periodic."};
-    static type default_value() { return make_array<1>(false); }
+    static type default_value() { return {{false}}; }
   };
   struct InitialRefinement {
     using type = std::array<size_t, 1>;
     static constexpr OptionString help = {"Initial refinement level in [x]."};
   };
 
   struct InitialGridPoints {
-    using type = std::array<size_t, 2>;
+    using type = std::array<std::array<size_t, 1>, 2>;
```

Comment:
This should probably be the other way around, with dimension the main index and segment secondary.

Reply:
As it's written, after indexing by the block we have an array indexed by dimension, which is most similar to how the other domaincreators are written

Reply:
True, but that doesn't generalize to higher dimensions here.  If we're going to have size-1 arrays they should be a special case of something with a larger size.

Reply:
okay, do something intelligent